bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -462,4 +462,46 @@
 		<state pattern="%s"/>
 	</channel-type>
 
+	<channel-type id="sfp-present">
+		<item-type>Switch</item-type>
+		<label>SFP Present</label>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="sfp-alim-ok">
+		<item-type>Switch</item-type>
+		<label>Alimentation Ok</label>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="sfp-has-power">
+		<item-type>Switch</item-type>
+		<label>Power Available</label>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="sfp-has-signal">
+		<item-type>Switch</item-type>
+		<label>Signal Present</label>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="link">
+		<item-type>Switch</item-type>
+		<label>Link is Active</label>
+		<category>Switch</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="sfp-signal-level">
+		<item-type unitHint="dBm">Number:Power</item-type>
+		<label>Signal Level</label>
+		<category>QualityOfService</category>
```

Comment:
Not sure. If I correctly remember, QualityOfService expects few discrete values to render the proper icon.

Reply:
The icon is not dynamic but properly rendered.

Reply:
Not fully convinced but ok let's keep this category.